nbs/directives.ipynb

```diff
@@ -0,0 +1,828 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "hide_input": false
+   },
+   "outputs": [],
+   "source": [
+    "#|default_exp directives"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#|hide\n",
+    "from fastcore.test import *\n",
+    "from nbdev.showdoc import show_doc"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Directives\n",
+    "\n",
+    "> The directives available in nbdev. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "Directives are special comments that have are preceeded by `#|` that do some kind of pre or post processing of a notebook data before docs are rendered. Some of these directives are part of Quarto, but others are unique to nbdev.\n",
+    "\n",
+    "See the [Quarto docs](https://quarto.org/docs/computations/execution-options.html) for even more execution options.\n",
```

Comment:
Directives in nbdev and quarto aren't just for execution options. Perhaps link to a couple of other places Quarto documents directives, and mention the types of things that quarto directives can change.

I'd also suggest writing a bit here about the fact that both nbdev and quarto use directives, and that all quarto directives can be used in an nbdev notebook.

Reply:
See comment: https://github.com/fastai/nbdev/pull/868#discussion_r948232459